recipes/mastercard/create_mastercard.sql

```diff
@@ -282,7 +282,7 @@ CREATE TEMP TABLE tmp_csv (
     yoy_txn_cnt text
 );
 
-\COPY tmp_csv from PSTDIN DELIMITER '|' CSV HEADER;
+\COPY tmp_csv from PSTDIN DELIMITER '|' CSV HEADER encoding 'UTF8';
```

Comment:
Why do we need encoding here?

Reply:
SQL is usually UTF8 by default. This makes it explicit. It is not necessary, and I forgot to remove it.